src/main/java/org/primeframework/mvc/security/CipherProvider.java

```diff
@@ -22,13 +22,13 @@
 import java.security.NoSuchAlgorithmException;
 
 /**
- * Provider for getting a Cipher instance that can be used for encryption and any other security as necessary.
+ * Provider for getting a Cipher instance that can be used for AES encryption and any other security as necessary.
  *
  * @author Brian Pontarelli
  */
 public interface CipherProvider {
```

Comment:
At the risk of word smithing here, is a split the middle between you guys then to rename the interface `BlockCipherProvider` or `IVBlockCipherProvider`? That would make it clear it makes assumptions about the IV. Breaking changes might make this a moot point though. One could argue `CipherProvider` shouldn't really be part of the public API for this library but that ship might have sailed.